src/applications/disability-benefits/781/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import Scroll from 'react-scroll';
+
+import { focusElement } from '../../../../platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
```

Comment:
Unit tests are typically included along with non-trivial components/logic. It would be good to include unit tests for this page, unless that is already planned in another PR.

Reply:
Thanks, yes, we have unit tests included in the following PR